### PR TITLE
convert default predicates to use the default

### PR DESCRIPTION
**Pull request by deads2k**
Builds on https://github.com/kubernetes/kubernetes/pull/50019 (lgtm'd already)

This converts the already default field selectors to use the default value.  I'll let CI point out the unit test failures for me to chase.

@kubernetes/sig-api-machinery-misc

### CODE REVIEW

File: pkg/registry/apps/controllerrevision/strategy_test.go [hunk 1]
```diff
@@ -127,55 +125,6 @@ func TestStrategy_ValidateUpdate(t *testing.T) {
...
-	}
-	if fieldSet.Get("metadata.namespace") != rev.Namespace {
-		t.Errorf("expeted %s found %s", rev.Namespace, fieldSet.Get("metadata.namespace"))
-	}
-	if labelSet.Get("foo") != rev.Labels["foo"] {
-		t.Errorf("expected %s found %s", rev.Labels["foo"], labelSet.Get("foo"))
-	}
-}
-
-func TestMatchControllerRevision(t *testing.T) {
```
Comment (sttts) [inline pkg/registry/apps/controllerrevision/strategy_test.go]: Don't we loose some test coverage here?

**Comment by sttts** [comment]
Went through this. Looks good. Crazy how often this code was repeated.

File: pkg/registry/apps/controllerrevision/strategy_test.go (hunk 1 above)
Comment (deads2k) [inline pkg/registry/apps/controllerrevision/strategy_test.go]: This entire method was removed.  I don't see a reason to re-test the stock selector.

Comment (sttts) [inline pkg/registry/apps/controllerrevision/strategy_test.go]: lgtm

**Comment by sttts** [comment]
/lgtm

Happy rebasing!

**Comment by deads2k** [comment]
rebased a boatload of files, but nothing controversial.

**Pull request merged**